pkg/ctl/update/cluster.go

```diff
@@ -82,6 +82,8 @@ func doUpdateClusterCmd(cmd *cmdutils.Cmd) error {
 		cfg.Metadata.Version = api.Version1_13
 	case api.Version1_13:
```

Comment:
This can also be:

```golang
case api.Version1_13, api.Version1_14:
```

but the cases aren't combined because when releasing a newer version, it'll help us remind of changing the `api.Version1_14` case.